core/types/deposit_tx.go

```diff
@@ -37,6 +37,8 @@ type DepositTx struct {
 	To *common.Address `rlp:"nil"`
 	// Mint is minted on L2, locked on L1, nil if no minting.
 	Mint *big.Int `rlp:"nil"`
+	// L1Cost is the L1 fee cost.
+	L1Cost *big.Int `rlp:"nil"`
```

Comment:
In general, we don't want to serialize the L1Cost into RLP, but may want to return it as part of the transaction JSON. However, for the specific case of the deposit TX, we don't have an L1 Fee.

Reply:
Removed.